config/db/db.go

```diff
@@ -35,6 +35,8 @@ func Init() error {
 	return nil
 }
 
+
```

Comment:
This whitespace should be removed.  Set up your editor to run `go fmt` after every save.